go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/lima-vm/lima
 
 go 1.23.0
 
+toolchain go1.24.1
+
```

Comment:
```suggestion
```

Reply:
It's an open issue dependabot/dependabot-core#11933

Reply:
Thanks! I suspect this will be fixed soonish, so I suggest we leave Go dependabot PRs open and ask dependabot to recreate them once the issue is fixed.

Reply:
I've decided to do manual fixes for now. I just ran

```
go mod edit -toolchain=none
git commit --amend -C HEAD
git push -f
```